utils/m0crate-io-conf

```diff
@@ -22,6 +22,7 @@ set -eu -o pipefail
 export PS4='+ [${BASH_SOURCE[0]##*/}:${LINENO}${FUNCNAME[0]:+:${FUNCNAME[0]}}] '
 
 PATH=$PATH:/opt/seagate/cortx/hare/bin
+xprt='libfab'
```

Comment:
How is this passed as an argument to the script? try bootstrapping with lnet and trying m0crate-io-conf.